http/src/main/resources/reference.conf

```diff
@@ -206,12 +206,10 @@ gke {
     namespaceNameSuffix = "namespace"
     services = [
       {
-        name = "galaxy-web"
+        name = "galaxy"
         kind = "ClusterIP"
       }
     ]
-    # Passed in galaxy config at helm install time
-    remoteUserName = "galaxy-user"
```

Comment:
I'm thinking we'll pass the real username as the Galaxy user since we have it anyway.